Formula/cwb3.rb

```diff
@@ -0,0 +1,102 @@
+class Cwb3 < Formula
+  desc "Tools for managing and querying large text corpora with linguistic annotations"
+  homepage "https://cwb.sourceforge.io/"
+  url "https://downloads.sourceforge.net/project/cwb/cwb/cwb-3.5-RC/cwb-3.4.33-src.tar.gz"
+  sha256 "856b72785522d42f13f4a0528d2b80c2bf422c10411234a8e4b61df111af77dd"
+  license "GPL-2.0-or-later"
+  head "svn://svn.code.sf.net/p/cwb/code/cwb/trunk"
+
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "pcre"
+  depends_on "readline"
+
+  uses_from_macos "bison" => :build
+  uses_from_macos "flex" => :build
+  uses_from_macos "ncurses"
+
+  resource("tutorial_data") do
+    url "https://cwb.sourceforge.io/files/encoding_tutorial_data.zip"
+    sha256 "bbd37514fdbdfd25133808afec6a11037fb28253e63446a9e548fb437cbdc6f0"
+  end
+
+  def install
+    args = %W[
+      PLATFORM=homebrew-formula
+      SITE=homebrew-formula
+      FULL_MESSAGES=1
+      PREFIX=#{prefix}
+      HOMEBREW_ROOT=#{HOMEBREW_PREFIX}
+    ]
+
+    system "make", "all", *args
+    ENV.deparallelize
+    system "make", "install", *args
+  end
+
+  def post_install
+    # make sure default registry exists
+    (HOMEBREW_PREFIX/"share/cwb/registry").mkpath
+  end
+
+  def caveats
+    default_registry = HOMEBREW_PREFIX/"share/cwb/registry"
```

Comment:
```suggestion
  def default_registry
    HOMEBREW_PREFIX/"share/cwb/registry"
  end

  def post_install
    # make sure default registry exists
    default_registry.mkpath
  end

  def caveats
```
Let's not hardcode this in two places.

Reply:
Excellent, thanks.